by-language/java-qa/src/test/java/io/crate/qa/JdbcMetaDataTest.java

```diff
@@ -191,6 +191,8 @@ public void test_getDatabaseProductVersion() throws Exception {
     }
 
     @Test
+    @Ignore("Not supported by CrateDB after pgjdbc 42.7.0 changed the implementation")
+    // https://github.com/crate/crate/issues/15113
```

Comment:
This mirrors the same skipping like applied on crate-qa.

- https://github.com/crate/crate-qa/pull/288